modules/cnv-enabling-dedicated-resources.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_virtual_machines/cnv_advanced_vm_management/cnv-dedicated-resources-vm.adoc
+// * cnv/cnv_vm_templates/cnv-dedicated-resources-vm-template.adoc
+
+// Establishing conditionals so content can be re-used for editing VMs
+// and VM templates.
+
+ifeval::["{context}" == "cnv-dedicated-resources-vm-template"]
+:cnv-vm-template:
+:object: virtual machine template
+:object-gui: Virtual Machine Template
+endif::[]
+
+ifeval::["{context}" == "cnv-dedicated-resources-vm"]
+:cnv-vm:
+:object: virtual machine
+:object-gui: Virtual Machine
+endif::[]
+
+[id="cnv-enabling-dedicated-resources_{context}"]
+
+= Enabling dedicated resources for a {object}
+
+ifdef::cnv-vm-template[]
+Use this procedure to enable dedicated resources and schedule workloads for virtual
+machine templates.
+endif::cnv-vm-template[]
+ifdef::cnv-vm[]
+Use this procedure to enable dedicated resources and schedule workloads for virtual
+machines.
+endif::cnv-vm[]
+
+.Procedure
+
+You can enable dedicated resources for a {object} in the
```

Comment:
The intro for a procedure goes before the procedure. 
You can replace lines 25-32 with this one (because it uses variables to differentiate between VMs and Templates)

Reply:
Done.